Code/GraphMol/RGroupDecomposition/RGroupMatch.h

```diff
@@ -6,8 +15,8 @@ namespace RDKit {
 typedef boost::shared_ptr<RGroupData> RData;
 typedef std::map<int, RData> R_DECOMP;
 struct RGroupMatch {
-  // RGroupMatch is the decomposition for a single molecule
-  size_t core_idx;   // index of the matching core
+  //! RGroupMatch is the decomposition for a single molecule
```

Comment:
I think if the idea is to document the struct, this comment should go before the definition. Still, this is internal-use code, so it's not super important

Reply:
I'll make the change and then commit.